scripts/CBnormalize.R

```diff
@@ -2,7 +2,7 @@ library(argparse)
 library(prismSeqR) # Is the package updated?
```

Comment:
what's the deal with this line? Can we remove it if we don't use prismSeqR?

Reply:
looks like prismSeqR is being loaded in multiple files too